ykbuild/build.rs

```diff
@@ -0,0 +1,51 @@
+use cmake;
+use std::env;
+use num_cpus;
+use which::which;
+
+const YKLLVM: &str = "../ykllvm/llvm";
+const PROFILE: &str = "Release";
+
+fn main() {
+    if env::var("YKB_YKLLVM_INSTALL_DIR").is_ok() {
+        println!("cargo:ykllvm={}", env::var("YKB_YKLLVM_INSTALL_DIR").unwrap());
+        return;
+    }
+
+    // Ninja builds are faster, so use this if it's available.
+    let use_ninja = which("ninja").is_ok();
+    let is_debug = env::var("PROFILE").unwrap() == "debug";
+    let nprocs = format!("-j {}", num_cpus::get());
+
+    let mut ykllvm = cmake::Config::new(YKLLVM);
+    ykllvm.profile(PROFILE)
+        .generator(if use_ninja { "Ninja" } else { "Unix Makefiles" })
+        .define("LLVM_INSTALL_UTILS", "ON")
+        .define("LLVM_ENABLE_PROJECTS", "lld;clang")
+        .define("LLVM_ENABLE_ASSERTIONS", if is_debug { "ON" } else { "OFF" })
```

Comment:
This needs `-DBUILD_SHARED_LIBS=ON`.

See https://github.com/ykjit/ykllvm/pull/67.

Reply:
Fixed 4dc0284131b98b7c2641863a68b73adde28d6c39